CHANGELOG.md

```diff
@@ -29,9 +33,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+* Allow commas in box titles
```

Comment:
"Allow commas…" and "Display correct minus sign…" were part of 0.6.

Reply:
Hmmm, looks like I picked up a few items from the transition point. I'll edit these back out.

Reply:
Done.